contracts/BondStorage.sol

```diff
@@ -332,7 +344,7 @@ contract BondStorage is AccessControl {
     }
 
     // Claims the payout in TST tokens by sending it to the user's wallet and resetting the claim to zero.
-    function claimReward(address _user) external {
+    function claimReward(address _user) external onlyWhitelisted {
```

Comment:
we don't need to protect anyone from claiming neither their own nor anyone else's reward since it will be transferred to the wallet with an actual reward pending